admin/perflab-module-migration-notice.js

```diff
@@ -0,0 +1,40 @@
+( function( document ) {
+	document.addEventListener( 'DOMContentLoaded', function() {
+		document.addEventListener( 'click', function(event) {
+			if ( event.target.classList.contains( 'perflab-install-active-plugin' ) ) {
+
+				if ( confirm( perflab_module_migration_notice.prompt_message ) ) {
```

Comment:
I don't think we need the confirm prompt. This isn't a destructive action, and having a confirm prompt makes it sound more alarming than it needs to be - particularly since we _want_ users to perform this action.

This will furthermore allow us to remove the `prompt_message` data.